src/SimpleList/index.js

```diff
@@ -229,23 +229,60 @@ class Row extends Component {
     return false
   }
 
+  renderImageLink(styleType) {
+    let { leftSection } = this.props
+    
+    let source = leftSection.image
```

Comment:
```suggestion
    const source = leftSection.image
```
Should be a const since it's not changing. Same with the `leftSection` variable up above.